tools/generators/pbxproject_prefix/src/Generator/BuildSettings.swift

```diff
@@ -0,0 +1,119 @@
+            "BAZEL_CONFIG": "rules_xcodeproj",
+            "BAZEL_EXTERNAL": "$(BAZEL_OUTPUT_BASE)/external".pbxProjEscaped,
+            "BAZEL_INTEGRATION_DIR": "$(INTERNAL_DIR)/bazel".pbxProjEscaped,
+            "BAZEL_LLDB_INIT":
+                "$(HOME)/.lldbinit-rules_xcodeproj".pbxProjEscaped,
+            "BAZEL_OUT": "$(PROJECT_DIR)/bazel-out".pbxProjEscaped,
+            "BAZEL_OUTPUT_BASE":
+                "$(_BAZEL_OUTPUT_BASE:standardizepath)".pbxProjEscaped,
+            "BAZEL_WORKSPACE_ROOT": "$(SRCROOT)".pbxProjEscaped,
+            "BUILD_DIR":
+                "$(SYMROOT)/$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)"
+                .pbxProjEscaped,
+            "BUILD_WORKSPACE_DIRECTORY": "$(SRCROOT)".pbxProjEscaped,
+            "BUILT_PRODUCTS_DIR":
+                "$(INDEXING_BUILT_PRODUCTS_DIR__$(INDEX_ENABLE_BUILD_ARENA))"
+                .pbxProjEscaped,
+            "CLANG_ENABLE_OBJC_ARC": "YES",
+            "CLANG_MODULES_AUTOLINK": "NO",
+            "CONFIGURATION_BUILD_DIR": "$(BUILD_DIR)/$(BAZEL_PACKAGE_BIN_DIR)"
+                .pbxProjEscaped,
+            "COPY_PHASE_STRIP": "NO",
+            "CURRENT_EXECUTION_ROOT":
+                "$(INDEXING_PROJECT_DIR__$(INDEX_ENABLE_BUILD_ARENA))"
+                .pbxProjEscaped,
+            "DEBUG_INFORMATION_FORMAT": "dwarf",
+            "DEPLOYMENT_LOCATION":
+                "$(INDEXING_DEPLOYMENT_LOCATION__$(INDEX_ENABLE_BUILD_ARENA))"
+                    .pbxProjEscaped,
+            "DSTROOT": "$(PROJECT_TEMP_DIR)".pbxProjEscaped,
+            "ENABLE_DEFAULT_SEARCH_PATHS": "NO",
+            "ENABLE_STRICT_OBJC_MSGSEND": "YES",
+            "GCC_OPTIMIZATION_LEVEL": "0",
+            "INDEXING_BUILT_PRODUCTS_DIR__":
+                "$(INDEXING_BUILT_PRODUCTS_DIR__NO)".pbxProjEscaped,
+            "INDEXING_BUILT_PRODUCTS_DIR__NO": "$(BUILD_DIR)".pbxProjEscaped,
+            "INDEXING_BUILT_PRODUCTS_DIR__YES":
+                "$(CONFIGURATION_BUILD_DIR)".pbxProjEscaped,
+            "INDEXING_DEPLOYMENT_LOCATION__":
+                "$(INDEXING_DEPLOYMENT_LOCATION__NO)".pbxProjEscaped,
+            "INDEXING_DEPLOYMENT_LOCATION__NO": "YES",
+            "INDEXING_DEPLOYMENT_LOCATION__YES": "NO",
+            "INDEXING_PROJECT_DIR__":
+                "$(INDEXING_PROJECT_DIR__NO)".pbxProjEscaped,
+            "INDEXING_PROJECT_DIR__NO": "$(PROJECT_DIR)".pbxProjEscaped,
+            "INDEXING_PROJECT_DIR__YES": indexingProjectDir.pbxProjEscaped,
+            "INDEX_FORCE_SCRIPT_EXECUTION": "YES",
+            "INSTALL_PATH":
+                "$(BAZEL_PACKAGE_BIN_DIR)/$(TARGET_NAME)/bin".pbxProjEscaped,
+            "INTERNAL_DIR": "$(PROJECT_FILE_PATH)/rules_xcodeproj",
+            "LD_DYLIB_INSTALL_NAME": "".pbxProjEscaped,
+            "LD_OBJC_ABI_VERSION": "".pbxProjEscaped,
+            "LD_RUNPATH_SEARCH_PATHS": "(\n\t\t\t\t)",
+            "ONLY_ACTIVE_ARCH": "YES",
+            // TODO: Set `RESOLVED_REPOSITORIES`
```

Comment:
We can't do this yet, we need input from the `files_and_groups` generator that will be made later.